frontend/src/authentication/components/logout/LogOutPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { logoutUser } from './../../containers/logoutUser.js';
+
+class LogOutPage extends React.Component {
+    componentWillMount() {
+        this.props.logoutUser();
```

Comment:
Not sure we need this page at all. You can have simple just logout action which redirects back to root of the application (which should render login page since user is not authenticated anymore)